src/Symfony/Component/Notifier/Bridge/OvhCloud/Tests/OvhCloudTransportTest.php

```diff
@@ -44,4 +48,42 @@ public function unsupportedMessagesProvider(): iterable
         yield [new ChatMessage('Hello!')];
         yield [$this->createMock(MessageInterface::class)];
     }
+
+    public function messagesProvider(): iterable
+    {
+        yield ['hello'];
+        yield ['hel/lo'];
+    }
+
+    /**
+     * @dataProvider messagesProvider
+     */
+    public function testSendWithNotification(string $message): void
```

Comment:
@derrabus shall we mark this test timesensitive?

Reply:
I think so, yes.